.github/labeler.yml

```diff
@@ -1,8 +1,27 @@
-version: 1
+version: 2
 labels:
+# type 1: package-action labels
 - label: "Update Package"
   title: '^upd\(.*'
+- label: "Update Pacscript"
+  title: '^fix\(.*'
+- label: "bug"
+  title: '^bug\(.*'
+- label: "enhancement"
+  title: '^enh\(.*'
+- label: "revert"
+  title: '^rev\(.*'
+
+# type 2: repo-action labels
 - label: "Package Add"
   title: "^add:.*"
 - label: "remove package"
   title: "^rm:.*"
+- label: "chore"
+  title: "^chore:.*"
+- label: "CI"
+  title: "^ci:.*"
+- label: "meta"
+  title: "^meta:.*"
+- label: "security"
+  title: "^sec:.*"
```

Comment:
```suggestion
```
If the security fix is in the pacscript, maintainers would use fix, and if it's an upstream fix, it'd be an upd.